webapp/.nvmrc

```diff
@@ -1 +1 @@
-v16.1.0
+v16.3.0
```

Comment:
@cpanato should workflow files for github actions [like this one](https://github.com/mattermost/focalboard/blob/main/.github/workflows/ci.yml#L32) that install node be also updated to use the new version?

Reply:
sgtm